srcDoc/release_notes.rst

```diff
@@ -67,6 +67,26 @@ Build HTML and PDF documentation (#94)
 RTI Perftest build script for linux now offers the option to generate the HTML
 and PDF documentation from the rst files in srcDoc.
 
+New parameter that allow set thread priorities (#65)
```

Comment:
RTI Perftest thread priorities can be configured via command-line parameter (#65)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The new parameter `-threadPriorities` has been added to *RTI Perftest* to allow
the user to set the priorities on the different threads created by *RTI Connext DDS*
and by the application.

This parameter accepts 3 numeric values which will be used as follows:

::

-threadPriorities X:Y:Z

Where:

- **X** is the priority for the main Thread that manages all the communication.
- **Y** is the priority for all the receive threads. This value will be used for the
  receive thread created by *RTI Connext DDS*, but also for the thread in charge
  of receiving the data when the `-useReadThread` option (use Waitsets) is provided.
- **Z** is the priority for the Event and DataBase threads created at the
  *RTI Connext DDS* level.

Reply:
Also add:

To see what values to be used for the threads See *RTI Connext DDS* Platform Notes Document, Section <add here section>

Reply:
I believe you are also missing mentioning that X will be used for the asynchronous thread when using large data. You might need to add that to the logic if you didn't.

Reply:
Done